### PR TITLE
Add Spanish translations

### PR DESCRIPTION
**Pull request by Lolabird**
A pull request to add Spanish translations to muffon

### CODE REVIEW

**Comment by xyloflake** [comment]
Hello @Lolabird! Thank you for the generous contribution, we really appreciate it.

However we have certain rules that **prohibit translations related PRs** to the repo.

The way to go with translations is to **join our discord server and request the translator role** as the preliminary steps and we'll take it from there. And don't worry, you'll get your credit in the README.MD file like every other translator did.

I'll have to close this as per our rules but we assure you, your work will not go unused, you just need to go ahead and join our discord server 😄

In case you're wondering why: it's both a **matter of legacy** (we've been doing this from the very start, so we try to maintain consistency across stuff) **and applications** (we need you when we add more features to the app, so we need a reliable way to contact you and the other translators together, which being in our case is discord). I hope you understand.

TL;DR: Your PR is being rejected because we have an existing mechanism for translations related contributions.

Warm regards,
Xylo on behalf of the muffon team
(Administrator and developer)

**Comment by Lolabird** [comment]
Hi @xyloflake, thanks for the info! I joined the Discord. Is there anything specific I need to do to get the translator role, or anywhere specific I can look for more info on how that works?

**Review by staniel359 — APPROVED** [review]

**Review by xyloflake — APPROVED** [review]
